VideoEncoding/Encoding_HEVC_ContentAware/Program.cs

```diff
@@ -1,650 +1,529 @@
-                    Console.WriteLine("Polling job status...");
-                    job = await WaitForJobToFinishAsync(client, config.ResourceGroup, config.AccountName, ContentAwareTransform, jobName);
-                }
-                finally
+Console.WriteLine();
+Console.WriteLine("Getting the streaming manifest URLs for HLS and DASH:");
+await PrintStreamingUrlsAsync(streamingLocator, streamingEndpoint);
+
+Console.WriteLine("To try streaming, copy and paste the streaming URL into the Azure Media Player at 'http://aka.ms/azuremediaplayer'.");
+Console.WriteLine("When finished, press ENTER to cleanup.");
+Console.WriteLine();
+Console.ReadLine();
+
+await CleanUpAsync(transform, job, inputAsset, outputAsset, streamingLocator, stopStreamingEndpoint, streamingEndpoint);
+
+#region EnsureTransformExists
+/// <summary>
+/// If the specified transform exists, return that transform. If the it does not
+/// exist, creates a new transform with the specified output. In this case, the
+/// output is set to encode a video using a custom preset.
+/// </summary>
+/// <param name="mediaServicesAccount">The Media Services account.</param>
+/// <param name="transformName">The transform name.</param>
+/// <returns>The transform.</returns>
+static async Task<MediaTransformResource> CreateTransformAsync(MediaServicesAccountResource mediaServicesAccount, string transformName)
+{
+    Console.WriteLine("Creating a Transform...");
+
+    // Create the custom Transform with the outputs defined above
+    // Does a Transform already exist with the desired name? This method will just overwrite (Update) the Transform if it exists already. 
+    // In production code, you may want to be cautious about that. It really depends on your scenario.
+    var transform = await mediaServicesAccount.GetMediaTransforms().CreateOrUpdateAsync(
+        WaitUntil.Completed,
+        transformName,
+        new MediaTransformData
+        {
+            Outputs =
+            {
+                // Create a new TransformOutput with a custom Standard Encoder Preset using the HEVC (H265Layer) codec
+                // This demonstrates how to create custom codec and layer output settings
+                new MediaTransformOutput(
+                    preset: new StandardEncoderPreset(
+                        codecs: new MediaCodecBase[]
+                        {
+                            // Add an AAC Audio layer for the audio encoding
+                            new AacAudio
+                            {
+                                Channels = 2,
+                                SamplingRate = 48000,
+                                Bitrate = 128000,
+                                Profile = AacAudioProfile.AacLc
+                            },
+                            // Next, add a HEVC (H.265) for the video encoding
+                            new H265Video
```

Comment:
This code does not setup Content Aware Encoding for HEVC., but a custom preset...

Reply:
I've updated the code to resolve this issue.